config.def.h

```diff
@@ -87,6 +87,17 @@ static float maxlatency = 33;
  */
 static unsigned int su_timeout = 200;
 
+/*
+ * Autoscroll interval when use mouse to select
+ */
+static unsigned int autoscrolltimeout = 200;
+
+/*
+ * After enable it, the speed of autoscroll is not be affected by the distance 
+ * betwen cursor and window.
+ */
+static unsigned int disable_autoscroll_accelerate = 0;
+
```

Comment:
Again, this is just a suggestion. I prefer that variables that are related to each other are named the same way. So this could be:

```suggestion
/*
 * When enabled, the speed of autoscroll is affected by the distance between
 * the mouse pointer and the window.
 */
static unsigned int autoscrollacceleration = 1;
```

But if you change it like this, then also note that its meaning changes to the opposite and that must be taken into account in the code.

But if you prefer the current variable name, then leave it be.

Reply:
fix